core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -185,9 +185,9 @@ final class JdbcUtil {
           + NAMESPACE_NAME
           + " VARCHAR(255) NOT NULL,"
           + NAMESPACE_PROPERTY_KEY
-          + " VARCHAR(5500),"
+          + " VARCHAR(255),"
```

Comment:
CATALOG_NAME + NAMESPACE_NAME + NAMESPACE_PROPERTY_KEY make up the primary key, and the primary key size is limited by 767 ( 255 * 3 = 765 < 767)